tests/models/llava_next/test_modeling_llava_next.py

```diff
@@ -208,14 +207,20 @@ def create_and_check_llava_next_model_fp16_autocast_forward(
 
 
 @require_torch
-class LlavaNextForConditionalGenerationModelTest(ModelTesterMixin, GenerationTesterMixin, unittest.TestCase):
+class LlavaNextForConditionalGenerationModelTest(ModelTesterMixin, unittest.TestCase):
```

Comment:
Why remove `GenerationTesterMixin`? I'm guessing because before `all_generative_model_classes` wasn't defined and this is multimodal but just want to make sure I've understood the change properly